rllib/core/rl_trainer/tf/tf_rl_trainer.py

```diff
@@ -151,7 +148,7 @@ def update(self, batch: MultiAgentBatch) -> Mapping[str, Any]:
     def compute_gradients(
         self, loss: Union[TensorType, Mapping[str, Any]], tape: tf.GradientTape
     ) -> ParamDictType:
-        grads = tape.gradient(loss["total_loss"], self._params)
+        grads = tape.gradient(loss[self.TOTAL_LOSS_KEY], self._params)
```

Comment:
What if `loss` is just a single tensor as declared in the typedef?

Reply:
I think that assumption of if loss is a tensor is being trumped. In some PRs later down the line, we will take another pass to revisit the assumptions that we originally made and make revisions.